tachyon/math/base/rings.h

```diff
@@ -8,9 +8,21 @@
 
 namespace tachyon::math {
 
+// Ring is a set S with operations + and * satisfying the followings:
```

Comment:
satisfying -> that satisfies. A slight grammar error here.

Reply:
I don't think the original one is incorrect, but I can change though if you prefer the way you suggested.